tests/test_browser.py

```diff
@@ -2212,7 +2212,11 @@ def test_glerror(self):
     self.btest('gl_error.c', expected='1', args=['-s', 'LEGACY_GL_EMULATION=1', '-lGL'])
 
   def test_openal_error(self):
-    for args in [[], ['--closure', '1']]:
+    for args in [
+      [],
+      ['-lopenal', '-s', 'STRICT'],
+      ['--closure', '1']
+    ]:
```

Comment:
I'm not sure I understand.  How does this works without `-lopenal` in the other cases?

Reply:
@sbc100 In non-strict mode, JS libraries are auto-included.